features/support/env.rb

```diff
@@ -38,19 +38,51 @@ def current_ip
   ip_list.captures.first
 end
 
-
-AfterConfiguration do |config|
-  install_fixture_gems
+def using_maze_runner_v7?
+  defined?(Maze)
```

Comment:
I guess this does the job, but you could also reference `Maze::VERSION` explicitly:

```
module Maze
  VERSION = '3.7.4'.freeze
end
```

and

```
module Maze
  VERSION = '7.5.1'
```

Reply:
Ah yeah, that would have been clearer! I think I'll leave it like this since after everything is upgraded it will get deleted anyway

Reply:
Yep, no problem.